### PR TITLE
Show quantity Under Journal tab for section purchased

### PR DESCRIPTION
**Pull request by rigidsphp**
Currently Reports Do not show how many items were purchased in a transaction. Even If 2 items of same product is purchased , Under Reports > Journal Role it just list 1 as purchased item. This update should show number of items in a single order

### CODE REVIEW

File: modules/report/includes/commerce_pos_report.pages.inc [hunk 1]
```diff
@@ -884,14 +884,22 @@ function commerce_pos_report_build_journal_role_table(array $filters = array())
 
       $return_count = 0;
       $purchase_count = 0;
-
+	  $number_of_products = 0;
+	  
+	  $order_wrapper = entity_metadata_wrapper('commerce_order', $row->order_id);
```
Comment (wilderuncat) [inline modules/report/includes/commerce_pos_report.pages.inc]: This should probably be moved into the switch statement below so it only gets triggered for purchases and not returns. Also the spacing needs some fixing.

Comment (rigidsphp) [reply modules/report/includes/commerce_pos_report.pages.inc]: By spacing you mean tab spacing or line spacing ? And yes that code should be under switch statement. Thank you

Comment (smmccabe) [reply modules/report/includes/commerce_pos_report.pages.inc]: yah, tabs should be 2 spaces to match drupal formatting. It helps if you setup phpcs within phpstorm.

https://confluence.jetbrains.com/display/PhpStorm/PHP+Code+Sniffer+in+PhpStorm
https://www.drupal.org/node/1419988

File: modules/report/includes/commerce_pos_report.pages.inc [hunk 2]
```diff
@@ -884,14 +884,21 @@ function commerce_pos_report_build_journal_role_table(array $filters = array())
 
       $return_count = 0;
       $purchase_count = 0;
-
+	  $number_of_products = 0;
```
Comment (wilderuncat) [inline modules/report/includes/commerce_pos_report.pages.inc]: This variable isn't used anymore.

**Comment by rigidsphp** [comment]
This is how it looks like http://puu.sh/vHzCx/9719aaf946.png on PHP Storm http://puu.sh/vHzCx/9719aaf946.png or may be loading my code in php storm corrected spacing itself ? So I can confirm tabs are 2 spaces only

**Comment by wilderuncat** [comment]
It's possible that loading it corrected it, mine does that for me. Check that your PHPStorm settings are set to use spaces instead of tabs: https://snag.gy/FqcJtW.jpg

**Comment by rigidsphp** [comment]
I believe The approach I followed earlier is not very optimized for performance , I cross checked database and found we might not need the for loop and we can correct the existing code by fetching the quantity field rather I will updating the patch again.

**Pull request closed without merging**